lib/core/kernel.nit

```diff
@@ -644,17 +644,23 @@ universal Byte
 	# `i` bits shift fo the left
 	#
 	#     assert 5u8 << 1    == 10u8
-	fun <<(i: Int): Byte `{ return self << i; `}
+	fun <<(i: Int): Byte is intern do return lsh(i)
+
+	private fun lsh(i: Int): Byte `{ return self << i; `}
```

Comment:
Yup, that's a fallback for any engine which does not support this implementation internally (coincidentally, this is only the interpreter for now :))

And no we can't for some reason, I tried but the FFI implementation would always be called, regardless whether or not there was an intern implementation for this.

Which leads us to the present form, not so cute, but it does the job pretty damn well